nexus-sdk-js/src/Project/utils.ts

```diff
@@ -127,3 +134,32 @@ export async function deprecateProject(
     throw new Error(error);
   }
 }
+
+export function subscribe(listeners: ProjectEventListeners): EventSource {
+  const event: EventSource = getEventSource('/projects/events');
+  //
+  // set event listeners
```

Comment:
Wrong comment style

Reply:
Fixed in 394299cab09907ae0f3c434bd34f10188b29a7a9